README.md

```diff
@@ -55,6 +55,13 @@ reconnect();
 > For trying to make a reconnection every 5 seconds if the connection is lost.
 
 ```
+ubidotsSetBusiness(bool business);
```

Comment:
This method should not go, with the setBroker or or setHost method should be sufficient